index.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
 	<title>Home</title>
+	<meta name="viewport" content="minimum-scale=1.0,user-scalable=no,initial-scale=1.0,maximum-scale=1.0,width=device-width">
```

Comment:
This tag tells the browser to not zoom the page out to fit all of the content on the screen. Instead, it will keep the page zoomed at 100% and only show as many pixels as the device screen has. You can red more here:

https://developer.mozilla.org/en-US/docs/Mozilla/Mobile/Viewport_meta_tag